ocis-pkg/keycloak/types.go

```diff
@@ -34,6 +34,8 @@ type PIIReport struct {
 type Client interface {
 	CreateUser(ctx context.Context, realm string, user *libregraph.User, userActions []UserAction) (string, error)
 	SendActionsMail(ctx context.Context, realm, userID string, userActions []UserAction) error
+	GetUserByParams(ctx context.Context, realm string, params gocloak.GetUsersParams) (*libregraph.User, error)
```

Comment:
Is this needed in the interface? For me this an internal function

Reply:
Yes and no, yes it's mainly used internally, but I can see it being useful for some people.